core/electron/package.json

```diff
@@ -39,7 +39,7 @@
     "@itwin/core-bentley": "workspace:^4.6.0-dev.13",
     "@itwin/core-common": "workspace:^4.6.0-dev.13",
     "@itwin/core-frontend": "workspace:^4.6.0-dev.13",
-    "electron": ">=23.0.0 <30.0.0"
+    "electron": ">=23.0.0 <31.0.0"
```

Comment:
Is there a reason to support electron versions which are EOL? https://www.electronjs.org/docs/latest/tutorial/electron-timelines#timeline

26.0.0 is EOL
<img width="693" alt="image" src="https://github.com/iTwin/itwinjs-core/assets/78756012/8e9b10ac-06c2-4249-8d8f-4ba63a387e12">

Reply:
Sigh, right.